proto/marketplace.proto

```diff
@@ -17,12 +17,10 @@ service Market {
 }
 
 message GetOrdersRequest {
-    // Slot describe resuorces to search for
-    Slot slot = 1;
-    // OrderType describe a type of orders to search
-    OrderType orderType = 2;
+    // Order keeps slot and type for searching
+    Order order = 1;
```

Comment:
It's a bad habbit - you are breaking API. It's ok for now, but unacceptable after release. PTAL at this https://developers.google.com/protocol-buffers/docs/gotutorial#extending-a-protocol-buffer

Reply:
I understand that this will break API, but for now, we have no stable API at all. 
After the release, we will change proto-things with care.